### ISSUE TITLE
FR: Comparison of Policies, Smart Groups etc.

### ISSUE THREAD

**Issue by master-vodawagner** [jamf/JamfMigrator#40]
More of a feature request, but it would be really cool if the migrator were to offer a comparison between the 2 Jamf instances against thinks like Policies , Smart Groups etc.

Identify whether a policy exists on one but not the other.

**Comment by BIG-RAT** [comment]
Good idea indeed.  I have a co-worker that had asked the same a while back so perhaps it's time to bump it up the list.  Policies though will be tricky as you can have duplicate display names.
